### PR TITLE
Deploy Kafka cleaner dashboards

### PR DESCRIPTION
**Pull request by welansari**
Issue: ZENKO-4742

### CODE REVIEW

**Review by KillianG — APPROVED** [review]

File: VERSION [hunk 1]
```diff
@@ -1,4 +1,4 @@
-VERSION="2.6.46"
+VERSION="2.6.47"
```
Comment (francoisferrand) [inline VERSION]: not sure it is worth releasing now: best to focus on getting the "full" 2.6.47 ready

**Review by francoisferrand — APPROVED** [review]

**Pull request merged**